### PR TITLE
ARC-1571: Add encrypt and decrypt on cryptor check

### PR DESCRIPTION
**Pull request by gxueatlassian**
**What's in this PR?**
1. Add an encrypt/decrypt check for Worker group in healthcheck. 
2. Add encrypt/decrypt check for WebServer in deepcheck.

**Why**
It is complicated, please refer to the ticket ARC-1571 itself to see a more detail doc page.

**Added feature flags**
N/A

**Affected issues**  
https://softwareteams.atlassian.net/browse/ARC-1571

**How has this been tested?**  
Stage (WIP)

**Whats Next?**
N/A

### CODE REVIEW

File: src/routes/healthcheck/healthcheck-get.ts [hunk 1]
```diff
@@ -1,8 +1,22 @@
...
 
 const logger = getLogger("healthcheck");
-export const HealthcheckGet = (_: Request, res: Response): void => {
+export const HealthcheckGet = async (_: Request, res: Response): Promise<void> => {
+
+	if (envVars.MICROS_GROUP === "Worker") {
+		try {
+			await deepCheckCryptor();
+		} catch (e) {
+			logger.warn("Cryptor not ready yet", { error: e });
```
Comment (joshkay10) [inline src/routes/healthcheck/healthcheck-get.ts:13]: `Cryptor not ready yet` could be confusing if its a legit Cryptor error, once it is ready.

Comment (gxueatlassian) [reply src/routes/healthcheck/healthcheck-get.ts:13]: I can't think of any way to distinguish whether it is just not ready or it is ready but has error. I can do a different text, like `Cryptor not ready yet or has error`? You reckon that's okay?

Comment (rachellerathbone) [reply src/routes/healthcheck/healthcheck-get.ts:13]: Can't we just use something more generic like `Cryptor deepcheck failed`? That plus the error should give enough info without being misleading.

Comment (gxueatlassian) [reply src/routes/healthcheck/healthcheck-get.ts]: thanks team, fixed per @rachellerathbone suggestion.

**Review by rachellerathbone — APPROVED** [review]

**Review by joshkay10 — APPROVED** [review]

**Pull request merged**